databuilder/databuilder/models/schema/schema_constant.py

```diff
@@ -11,4 +11,4 @@
 DATABASE_SCHEMA_KEY_FORMAT = '{db}://{cluster}.{schema}'
 
 # pattern used to match a schema key, e.g., hive://gold.test_schema
-SCHEMA_KEY_PATTERN_REGEX = '([a-z]+://[a-zA-Z0-9_-]+).[a-zA-Z0-9_.-]+'
+SCHEMA_KEY_PATTERN_REGEX = '([a-zA-Z0-9_]+://[a-zA-Z0-9_-]+).[a-zA-Z0-9_.-]+'
```

Comment:
the name uri is db://cluster.schema.table, db normally is hive, snowflake, presto etc. why the db will be sth else?

Reply:
I guess the reason is that we only have snowflake but not any other data warehouse, so initially we didn't include `snowflake` into keys. Instead of constructing keys like`{db=snowflake}://{cluster=snowflake_database}.{schema=schema_name}/{table=table_name}`, we are doing `{db=snowflake_database}://{cluster=production/staging, etc.}.{schema=schema_name}/{table=table_name}`.